### ISSUE TITLE
wav2vec2 not pre-training when using a 100h dataset.

### ISSUE THREAD

**Issue by DomainFlag** [pytorch/fairseq#3744]
#### What is your question?

I am trying to pre-train wav2vec2 on default LibriSpeech dataset (100h). I'm using the default `wav2vec2_base_librispeech.yaml` config but the training loss gets stuck always at 6.657.

#### What have you tried?

The only difference from the original config is the dataset size (from 960h -> 100h) and the `distributed_world_size` set to 1 as I have only one GPU, while the `update_freq` is set to 8 to compensate for the low number of GPUs that I have. As far as I know `update_freq` is for gradient accumulation to simulate training on 8 GPUs in my case. Does it work specifically only when simulating 64 GPUs, that is setting `update_freq` to 64? Or it's another issue as I don't what to test that much with `update_freq` to 64 as it will take a long way.

My objective for now is just to reproduce at least somewhat okayish embeddings that can be later on be decoded to somewhat discernible targets. 
 
#### What's your environment?

What's your environment?
fairseq Version (e.g., 1.0 or master): master
PyTorch Version (e.g., 1.0): 1.9.0
OS (e.g., Linux): Ubuntu 18.04 LTS
How you installed fairseq (pip, source): source
GPU models and configuration: 1x Nvidia P100

**Comment by jubick1337** [comment]
Hi @DomainFlag 
Recently I've faced the same issue 
After my investigation I can say that it's all about LR and update frequency
Lowering LR is going to solve freezing loss but it's not going to give you best loss (but okayish)

**Comment by DomainFlag** [comment]
@jubick1337 Hi, thank you for the response. Don't you mind sharing what config you've used with what LR (how low did you set it to), `distributed_world_size` and `update_freq` and on which dataset you had success with and any other changes besides that?

**Comment by jubick1337** [comment]
LR: 0.0001 update freq:4 Using 1-2 GPUs
I've also set batch_size to be higher as I'm using RTX 3090 which should give better convergence [Scaling Neural Machine Translation Myle Ott](https://arxiv.org/pdf/1806.00187.pdf)
Custom made Russian speech dataset (Don't think it matters though)

**Comment by DomainFlag** [comment]
@jubick1337 Thank you!
